ONSdigital/design-system
# Document form component validation error messages

### What is the context of this PR?
Updated the following components and patterns to include error validation documentation and examples:

#### Components
- Access codes
    - Also renamed 'uac' pattern folders to 'access-code' *** **BREAKING CHANGE** ***
- Checkboxes
    - Also updated all examples to be more realistic use cases and use `legendIsPageTitle`
- Dates
- Durations
- Input
    - Also changed prefix/suffix background colour to match secondary buttons
    - Also updated width-constrained examples to use correct number classes
- Mutually exclusive
- Radios
- Select
- Summary
- Textarea
- Upload

#### Patterns
- Dates
- Download resources
- Durations
    - Also removed unnecessary time example
- Email addresses
- Feedback
- Find a support centre
- Numeric values
- Send a web form

Also fixed any typos in the content, and adjusted heading levels for consistency across similar templates.
 
### How to review
- Check all examples and error documentation for all the patterns and components listed above. 
- Check there are no issues with access codes after the folder name change
